.travis.yml

```diff
@@ -0,0 +1,36 @@
+os: linux
+dist: bionic
+language: bash
+services: docker
+
+branches:
+  only:
+  - master
+
+env:
+  - VERSION=4.5
+  - VERSION=4.6
+  - VERSION=4.7
+
+jobs:
+  include:
+    - stage: Build x86_64 image
+      arch: amd64
+      script:
+        - docker build --build-arg RELEASE_VER=$VERSION -t openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-$(uname -m) -f images/Dockerfile .
```

Comment:
Please use the `https://quay.io/repository/powercloud/` org for the image and login to quay.io.